src/function_run_result.rs

```diff
@@ -23,18 +23,21 @@ pub struct FunctionRunResult {
     pub memory_usage: u64,
     pub instructions: u64,
     pub logs: String,
+    pub input: String,
     pub output: FunctionOutput,
     #[serde(skip)]
     pub profile: Option<String>,
 }
 
 impl FunctionRunResult {
+    #[allow(clippy::too_many_arguments)]
     pub fn new(
```

Comment:
Added this due to a `= note: `-D clippy::too-many-arguments` implied by `-D warnings`. Is this okay to do?